dev/com.ibm.ws.webcontainer/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -176,4 +176,7 @@ stopAppStartUponListenerException.name=Stop application for unhandled listener e
 stopAppStartUponListenerException.desc=Some web applications depend on context listeners for setup before the web application starts. When this property is set to true, the application stops starting up when an unhandled exception is thrown from the context listeners. For servlet-5.0 and newer Servlet features, the default value is true. 
 
 redirectToRelativeUrl.name=Send redirect response to a relative URL location
-redirectToRelativeUrl.desc=Send redirect response to a relative URL location without processing it. Set this property to true to send redirect response without converting the URL to an absolute location. 
\ No newline at end of file
+redirectToRelativeUrl.desc=Send redirect response to a relative URL location without processing it. Set this property to true to send redirect response without converting the URL to an absolute location. 
```

Comment:
suggested edit:
Send a redirect response to a relative URL location without processing the URL. Set this property to true to send a redirect response without converting the URL to an absolute location. 

I wonder if the first sentence is necessary? Kinda says the same thing twice